Common/Net/HTTPClient.cpp

```diff
@@ -329,7 +329,7 @@ int Client::SendRequestWithData(const char *method, const RequestParams &req, co
 	return 0;
 }
 
-int Client::ReadResponseHeaders(net::Buffer *readbuf, std::vector<std::string> &responseHeaders, net::RequestProgress *progress) {
+int Client::ReadResponseHeaders(net::Buffer *readbuf, std::vector<std::string> &responseHeaders, net::RequestProgress *progress, std::string * httpCode) {
```

Comment:
nit: Would prefer `statusLine` rather than `httpCode` which isn't very clear.  Also would prefer `*` next to the name, as is the convention used elsewhere.  It's nice to dream that C+ would make pointer part of the type, but it isn't.  `int* foo, bar` is not two pointers, so the `*` is objectively in the wrong place.

-[Unknown]

Reply:
i think VS2022 inserted that space when i copy-paste it while fixing the conflict during merge 🤔 because i never put spaces on both sides of '*' except for multiplication.

Reply:
Yeah, VS2022 seems to have some weird default formatting settings, like that ugly `else` formatting...

-[Unknown]